.github/workflows/test.yml

```diff
@@ -170,7 +171,31 @@ jobs:
 
         # Start mongo container and app before running api tests
         run: |
-          cp CI/ESS/docker-compose.api.yaml docker-compose.yaml
           cp functionalAccounts.json.test functionalAccounts.json
-          docker-compose up --build -d
-          npm run test:api
+          docker-compose up --build -d -f CI/ESS/docker-compose.api.yaml
```

Comment:
caution, this is inside a matrix run, so it will do concurrent api generation on both a elastic-search enabled BE and a plain backend (see line 105ff. )

I would suggest to move the api generation before that block ( and add a depends condition, so that the ordering is correct.